shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -50,6 +50,8 @@ FLUTTER_EXPORT
  * A newly initialized engine will not run the `FlutterDartProject` until either
  * `-runWithEntrypoint:` or `-runWithEntrypoint:libraryURI:` is called.
  *
+ * FlutterEngine created with this method will have allowHeadlessExecution set to YES.
```

Comment:
It might be better to explain what this does rather than the parameter it sets.

Something like

```
Creating a FlutterEngine with this initializer defaults allowHeadlessExecution to `YES`. This means that the engine will continue to run regardless of whether a `FlutterViewController` is attached to it or not.
```